### ISSUE TITLE
is the tokenization broken for bert?

### ISSUE THREAD

**Issue by AdityaSoni19031997** [huggingface/transformers#2136]
## 🐛 Bug

Model I am using is `bert-base-uncased`:

Language I am using the model on (English):

## To Reproduce

1. Just Ran the example from the docs

```
import torch
from transformers import BertTokenizer, BertModel, BertForMaskedLM

# OPTIONAL: if you want to have more information on what's happening under the hood, activate the logger as follows
import logging
logging.basicConfig(level=logging.INFO)

# Load pre-trained model tokenizer (vocabulary)
tokenizer = BertTokenizer.from_pretrained('bert-base-uncased', force_download=True)

# Tokenize input
text = "[CLS] Who was Jim Henson ? [SEP] Jim Henson was a puppeteer [SEP]"
tokenized_text = tokenizer.tokenize(text)

# Mask a token that we will try to predict back with `BertForMaskedLM`
masked_index = 8
tokenized_text[masked_index] = '[MASK]'
print(tokenized_text)
assert tokenized_text == ['[CLS]', 'who', 'was', 'jim', 'henson', '?', '[SEP]', 'jim', '[MASK]', 'was', 'a', 'puppet', '##eer', '[SEP]']

>INFO:transformers.tokenization_utils:loading file https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-vocab.txt from cache at /Users/1570137/.cache/torch/transformers/26bc1ad6c0ac742e9b52263248f6d0f00068293b33709fae12320c0e35ccfbbb.542ce4285a40d23a559526243235df47c5f75c197f04f37d1a0c124c32c9a084
```

The assertion fails and the `print(tokenized_text)` returns this actually,

`['[', 'cl', '##s', ']', 'who', 'was', 'jim', 'henson', '[MASK]', '[', 'sep', ']', 'jim', 'henson', 'was', 'a', 'puppet', '##eer', '[', 'sep', ']']`

## Extra Details

>! pip show transformers
```
Name: transformers
Version: 2.2.1
Summary: State-of-the-art Natural Language Processing for TensorFlow 2.0 and PyTorch
Home-page: https://github.com/huggingface/transformers
Author: Thomas Wolf, Lysandre Debut, Victor Sanh, Julien Chaumond, Google AI Language Team Authors, Open AI team Authors, Facebook AI Authors, Carnegie Mellon University Authors
Author-email: thomas@huggingface.co
License: Apache
Location: /Users/1570137/anaconda3/envs/my_env/lib/python3.7/site-packages
Requires: sacremoses, numpy, tqdm, sentencepiece, regex, requests, boto3
Required-by: flair
```

I am on MacOS, No GPU.
Also is that behaviour expected?
Thanks.

**Comment by BramVanroy** [comment]
This should be fixed in the current master but not in a release AFAIK. See https://github.com/huggingface/transformers/issues/2132 and close this issue please.

**Comment by AdityaSoni19031997** [comment]
Okay thanks!
